### PR TITLE
(POC, WIP) Extract Greenwich as an extension. Draft bootstrap.css variant.

### PR DESCRIPTION
**Pull request by totten**
Overview
----------------------------------------
_A brief description of the pull request. Keep technical jargon to a minimum. Hyperlink relevant discussions._

Before
----------------------------------------
_What is the old user-interface or technical-contract (as appropriate)?_
_For optimal clarity, include a concrete example such as a screenshot, GIF ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)), or code-snippet._

After
----------------------------------------
_What changed? What is new old user-interface or technical-contract?_
_For optimal clarity, include a concrete example such as a screenshot, GIF ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)), or code-snippet._

Technical Details
----------------------------------------
_If the PR involves technical details/changes/considerations which would not be manifest to a casual developer skimming the above sections, please describe the details here._

Comments
----------------------------------------
_Anything else you would like the reviewer to note_

### CODE REVIEW

File: ext/csslib/info.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  </maintainer>
+  <urls>
+    <url desc="Main Extension Page">http://civicrm.org</url>
+    <url desc="Documentation">http://civicrm.org</url>
+    <url desc="Support">http://civicrm.org</url>
+    <url desc="Licensing">http://www.gnu.org/licenses/agpl-3.0.html</url>
+  </urls>
+  <releaseDate>2020-07-21</releaseDate>
+  <version>1.0</version>
+  <develStage>alpha</develStage>
```
Comment (eileenmcnaughton) [inline ext/csslib/info.xml:19]: Let's keep an eye on this alpha & not leave it at that

**Comment by colemanw** [comment]
@totten could you please add some description to this PR?
I'm trying to figure out how to use it. I've applied the patch, enabled the extension, run `composer install` from within the `civicrm/ext/greenwich` directory. Now when I navigate to a bootstrap page like the APIv4 explorer... nothing different.
What else do I need to do to get bootstrap to load?

**Comment by eileenmcnaughton** [comment]
@colemanw is the theme selected on the display preferences page?

**Comment by colemanw** [comment]
> @colemanw is the theme selected on the display preferences page?

Yes, I tried setting both the backend and frontend themes to "Greenwich (with Bootstrap)"

**Comment by totten** [comment]
The approach is changing, so this PR is not needed.

**Pull request closed without merging**